docs/UserGuide.md

```diff
@@ -36,10 +36,28 @@ you can save it in the Personal Tracker provided by ExchangeCourseMapper!
 
 ## Features 
 
-> NOTE: Your stored course mapping is located in the `myList.json` file, found in the `data` folder at the same file path where you executed the JAR file.
+> **NOTE:** Your stored course mapping is located in the `myList.json` file, found in the `data` folder at the 
+> same file path where you executed the JAR file.
+>
+> **NOTE:** Ensure that all the commands given are placed in **one line**. 
+> Do not separate the command into multiple lines in the terminal.
 > 
-> NOTE:  Do not corrupt the `myList.json` file when executing the program! If you corrupted the file, please revert back to its original state before continuing with the program. 
- 
+> **NOTE:** For all command flags, ensure that your input is placed a single space after the flag.
+> 
+> **NOTE:** If the file is **corrupted**, please restore it to its **original state** before proceeding with the program.
+> 
+>You may try one of the following:
+>- Remove the corrupted line (delete one course mapping),
+>- Remove any empty lines in the file.
+>
+> **This list is non-exhaustive.**
+> 
+> **TIP:** You may use the partner universities' **official abbreviation** instead of typing the full name into the CLI, 
+> as shown in the chart below:
+> - The University of Melbourne - unimelb
+> - The Australian National University - anu
+> - Victoria University of Wellington - wgtn
+> - The University of Western Australia - uwa
```

Comment:
This is a clear way for users to know that they have another option of inputting the university names!